.github/workflows/codegen.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, ep20 ]
```

Comment:
**fix**: This branch should just be renamed and use a wildcard e.g. [feat-*](https://github.com/awslabs/aws-sdk-kotlin/blob/main/.github/workflows/continuous-integration.yml#L9). Same thing in smithy-go which was already merged.